gtkm/static/index.html

```diff
@@ -31,7 +32,7 @@ <h2>Let us introduce your<br><span>coding experience</span></h2>
                 eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad 
                 minim veniam, quis nostrud exercitation 
                 ullamco laboris nisi ut aliquip ex ea commodo consequat.</p>
-                <a class="github" href="http://127.0.0.1:8000/api/auth/github/authorize" target="_blank">Sign in with GitHub </a>
+                <a class="github" href="http://127.0.0.1:8000/api/auth/github/authorize">Sign in with GitHub </a>
```

Comment:
Absolutnie nigdzie w kodzie nie powinieneś używać takich linków z adresami i portem.

Reply:
Żebym dobrze zrozumiał - zmiana z #24 jest przede wszystkim po to, aby w razie zmiany adresu lub portu zmieniać go w jednym znanym miejscu, a nie po wszystkich możliwych plikach?

Reply:
Tak

Reply:
Poza tym w żadnej dobrze napisanej stronie nie zobaczysz linków podawanych w ten sposób.

Jako przykład możesz sobie nawet podejrzeć kod samego githuba.

```html
<a class="author Link--primary text-bold css-truncate-target " 
data-hovercard-type="user" 
data-hovercard-url="/users/kamilwil/hovercard" 
data-octo-click="hovercard-link-click" data-octo-dimensions="link_type:self" 
href="/kamilwil">kamilwil</a>
```
Nigdzie nie uświadczysz linka w stylu: `https://github.com/kamilwil`